.ci/bears.dirs.prune.sh

```diff
@@ -0,0 +1,13 @@
+# Delete empty bear directories
+set -e
+
+dirs=$(find bears -depth -type d -and -not -name '__pycache__')
+
+for dir in $dirs; do
+  bears=$(ls $dir/*Bear.py 2>/dev/null || true)
+  subdirs=$(ls -d $dir/*/ 2>/dev/null || true)
+  if [[ -z "$bears""$subdirs" ]]; then
+    echo Removing $dir
+    rm -rf $dir
+  fi
```

Comment:
Why is it important to delete the empty directories? What if we left them as it is?

Reply:
I forget.

probably another bear will complain about a missing `__init__.py`, or we have missing coverage of those `__init__.py`